cirq-core/cirq/sim/act_on_density_matrix_args_test.py

```diff
@@ -92,3 +113,18 @@ class NoDetails:
     )
     with pytest.raises(TypeError, match="Can't simulate operations"):
         cirq.act_on(NoDetails(), args, qubits=())
+
+
+def test_with_qubits():
+    original = cirq.ActOnDensityMatrixArgs(
+        initial_state=1,
+        qubits=cirq.LineQubit.range(1),
+    )
+    extened = original.with_qubits(cirq.LineQubit.range(1, 2))
+    np.testing.assert_almost_equal(
+        extened.target_tensor,
+        transformations.density_matrix_kronecker_product(
```

Comment:
I believe this function is in `cirq` module itself, so don't need to import `transformations` explicitly. (Generally tests don't have to import anything but `cirq` from cirq.)

Reply:
But pytest reports errors when I try it. I have added this function to the `__init__.py` files so that it would be included in `cirq` module.

Reply:
Thanks, you can resolve this one.